core/src/test/resources/logback-test.xml

```diff
@@ -20,7 +20,7 @@
 
     <root level="info">
         <appender-ref ref="FILE"/>
-        <appender-ref ref="console"/>
+        <appender-ref ref="console"/>
```

Comment:
not strictly part of this PR - these were previously [&ensp;](https://alibaba-cloud.medium.com/introduction-to-html-space-notation-nbsp-ensp-emsp-a6eacc98a0a7) and now they are standard spaces.